mqtt/mqtt-broker/src/ready.rs

```diff
@@ -95,7 +95,10 @@ pub trait AwaitingEvents {
 /// Broker readiness events.
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 pub enum BrokerReadyEvent {
+    /// Signals that `EdgeHubAuthorizer` is ready.
     AuthorizerReady,
+    /// Signals that `PolicyAuthorizer` is ready.
+    PolicyReady,
```

Comment:
What is the difference between `PolicyReady` and `AuthorizerReady`? Should we combine it into one event because they both related to the authorizer?

Reply:
Those are two distinct messages coming from EdgeHub. Later we may thing about optimizing this, but for now we need two events.

Reply:
Help me to understand the full flow of how these events originated in EHC and pushed to the broker. 
It starts with EdgeHub received its own twin (from storage or via wire). Then it publishes several messages to broker: 1) policy update; 2) configuration update; 3) ...?  
In parallel EdgeHub makes some call to obtain child devices scopes? When it receives them it publishes separate messages with device scopes. That's why we cannot get away with only one event right now?

Reply:
yes, policy and bridge config comes from the twin. Device scopes (authorized identities) comes from rest call to the IoT Hub. Those components can be available at different moments in time, so we can't combine them atm.